app/connectors/spv.js

```diff
@@ -4,6 +4,7 @@ import * as sel from "../selectors";
 
 const mapStateToProps = selectorMap({
   spvMode: sel.spvMode,
+  blocksNumberToNextTicket: sel.blocksNumberToNextTicket,
```

Comment:
You can drop the next item (`blocksPassedOnTicketInterval`) since it's not used in the component anymore